.gitignore

```diff
@@ -11,3 +11,4 @@ data/
 .idea/
 **/*.iml
 **/.DS_Store
+/vendor
```

Comment:
The git repository contains a checked in `vendor` directory. Are you sure this needs to be ignored?